src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -187,14 +187,23 @@ export default class CLWCreditsDaysLeft extends Component {
       <Card>
         <CardContent>
           <Grid container direction="row" alignItems="center">
-            <Grid item xs={7} align="left">
+            <Grid item xs={4} align="left">
+              <Button
+                variant="contained"
+                style={style.button}
+                onClick={(event) => (window.location.href = '/attended')}
```

Comment:
Rather than doing this browser navigation, using the browser's `window` object to change the URL, 360 uses a routing package called `react-router`. The main advantage is that, while browser navigation reloads the entire page from scratch, `react-router` just changes which component is displayed at the view level - i.e. instead of `Home` we render `Events`. This means we aren't reloading the entire site from scratch, so the user has a much quicker, much smoother experience.

If you import the `Link` component from the `react-router-dom` package at the top of this file with the other imports, then you can use it for the behavior of the button:
```suggestion
                component={Link}
                to="/attended"
```